src/queryResult/queryResultWebViewController.ts

```diff
@@ -309,4 +325,22 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
         const messageText = messages.join("\n");
         await this._vscodeWrapper.clipboardWriteText(messageText);
     }
+
+    public getNumExecutionPlanResultSets(
```

Comment:
Do you think we should add some unit tests for this new functionality?

Reply:
yes! that'll be my next pr